tmail-backend/integration-tests/jmap/memory-jmap-integration-tests/src/test/java/com/linagora/tmail/james/DavMemoryLinagoraCalendarEventAcceptMethodTest.java

```diff
@@ -0,0 +1,179 @@
+ *  https://www.gnu.org/licenses/agpl-3.0.en.html                   *
+ *                                                                  *
+ *  This program is distributed in the hope that it will be         *
+ *  useful, but WITHOUT ANY WARRANTY; without even the implied      *
+ *  warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR         *
+ *  PURPOSE. See the GNU Affero General Public License for          *
+ *  more details.                                                   *
+ ********************************************************************/
+
+package com.linagora.tmail.james;
+
+
+import static com.linagora.tmail.configuration.OpenPaasConfiguration.OPENPAAS_QUEUES_QUORUM_BYPASS_DISABLED;
+import static org.apache.james.data.UsersRepositoryModuleChooser.Implementation.DEFAULT;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.file.Path;
+import java.util.Optional;
+
+import jakarta.inject.Singleton;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.http.auth.UsernamePasswordCredentials;
+import org.apache.james.JamesServerBuilder;
+import org.apache.james.JamesServerExtension;
+import org.apache.james.core.Username;
+import org.apache.james.jmap.rfc8621.contract.probe.DelegationProbeModule;
+import org.apache.james.mailbox.MessageIdManager;
+import org.apache.james.mailbox.SessionProvider;
+import org.apache.james.mailbox.model.MessageId;
+import org.apache.james.utils.TestIMAPClient;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import com.github.fge.lambdas.Throwing;
+import com.google.common.collect.ImmutableList;
+import com.google.inject.AbstractModule;
+import com.google.inject.Module;
+import com.google.inject.Provides;
+import com.linagora.tmail.AmqpUri;
+import com.linagora.tmail.DockerOpenPaasExtension;
+import com.linagora.tmail.OpenPaasModuleChooserConfiguration;
+import com.linagora.tmail.OpenPaasUser;
+import com.linagora.tmail.api.OpenPaasRestClient;
+import com.linagora.tmail.configuration.DavConfiguration;
+import com.linagora.tmail.configuration.OpenPaasConfiguration;
+import com.linagora.tmail.dav.CalDavEventAttendanceRepository;
+import com.linagora.tmail.dav.DavClient;
+import com.linagora.tmail.dav.DavUserProvider;
+import com.linagora.tmail.dav.OpenPaasDavUserProvider;
+import com.linagora.tmail.james.app.MemoryConfiguration;
+import com.linagora.tmail.james.app.MemoryServer;
+import com.linagora.tmail.james.common.InvitationEmailData;
+import com.linagora.tmail.james.common.LinagoraCalendarEventAcceptMethodContract;
+import com.linagora.tmail.james.common.User;
+import com.linagora.tmail.james.jmap.EventAttendanceRepository;
+import com.linagora.tmail.james.jmap.firebase.FirebaseModuleChooserConfiguration;
+import com.linagora.tmail.module.LinagoraTestJMAPServerModule;
+
+
+public class DavMemoryLinagoraCalendarEventAcceptMethodTest implements
```

Comment:
The entire point of a module chooser is to easily turn on/off modules without the kind of ceremony in this class.